spec/mailers/user_mailer_spec.rb

```diff
@@ -137,6 +137,10 @@
       it 'only has style_to links in the HTML body' do
         expect(get_message_part(email, /html/)).not_to have_xpath('//a[not(@style)]')
       end
+
+      it 'does not have exposed HTML' do
```

Comment:
It would be a good idea to also add a test for the email not containing "translation missing" -- I see a few instances of it in the HTML email:
1. `You can read announcements about recent archive moves at <a href="http://archiveofourown.org/admin_posts?tag=18" style="color:#990000"><span class="translation_missing" title="translation missing: en.user_mailer.invitation_to_claim.html.ao3_news">Ao3 News</span></a>`
2. `Please just <a href="http://transformativeworks.org/contact/open%20doors" style="color:#990000"><span class="translation_missing" title="translation missing: en.user_mailer.invitation_to_claim.contact_open_doors">Contact Open Doors</span></a>`
3. `If you would like Open Doors to update the redirect to point to your pre-existing work, please delete the imported copy, and <a href="http://transformativeworks.org/contact/open%20doors" style="color:#990000"><span class="translation_missing" title="translation missing: en.user_mailer.invitation_to_claim.contact_open_doors">Contact Open Doors</span></a>`
4. `please <a href="http://transformativeworks.org/contact/open%20doors" style="color:#990000"><span class="translation_missing" title="translation missing: en.user_mailer.invitation_to_claim.contact_open_doors">Contact Open Doors</span></a> with any information that can help verify your identity.`